conf.yaml

```diff
@@ -291,20 +291,20 @@ contents:
             chunk:      1
             tags:       XPack/Reference
             current:    5.3
-            index:      docs/public/index.asciidoc
+            index:      docs/en/index.asciidoc
             private:    1
             branches:   [ 5.3, 5.2, 5.1, 5.0 ]
             sources:
               -
                 repo:   x-pack
-                path:   docs/public
+                path:   docs/en
               -
                 repo:   x-pack-kibana
-                path:   docs/public-kb
+                path:   docs/en
                 exclude_branches:   [ 5.3, 5.2, 5.1, 5.0, 2.4, 2.3, 2.2, 2.1, 2.0]
```

Comment:
`exclude_branches` should be a subset of `branches`, so you can delete all the 2.x branches